index.d.ts

```diff
@@ -578,6 +578,20 @@ export declare interface TracerOptions {
        * @default 'safe'
        */
       mode?: 'safe' | 'extended' | 'disabled'
+    },
+
+    /**
+     * Configuration for Api Security sampling
+     */
+    apiSecurity?: {
+      /** Whether to enable Api Security.
+       * @default false
+       */
+      enabled?: boolean,
+      /** Controls the request sampling rate (between 0 and 1) in which Api Security is triggered.
```

Comment:
Could use a little ligne explaining that values will be "coerced" back if outside the 0-1 range

Reply:
Actually, even I just realized, even if you remove the Math.min(Math.max()) thingy from the config, it would still have the same behavior no ?

Reply:
Yep but as we discussed in another comment it could be helpful in the future in we change how the sampling rate is computed.
Anyway I added the comment, thanks for noticing it.